### PR TITLE
Fixes for recent deploy

### LINKED ISSUE #644: App crashes during registration in Staging (version 5.6.1)
While I was testing registration in Staging, the App crashed at the credit card stage after I tried using the Dev buttons to move to the next step. It crashed using the two buttons circled below in blue (also, the thumbs up/thumbs down emojis on these buttons are missing!). No reaction when trying to use the button circled in red, or after swiping my credit card. 

img_0003

### LINKED ISSUE #645: Grid layout reacts abnormally when switching filters (version 5.6.1)
I saw this mostly when switching between "Grid" and "Least Bids." It occasionally goes back to normal if you scroll down the page and scroll back up. 

img_0004

### PR DESCRIPTION
**Pull request by ashfurrow**
Fixes #644. 
Fixes #645.

### CODE REVIEW

File: Kiosk/Bid Fulfillment/StripeManager.swift [hunk 1]
```diff
@@ -10,10 +10,10 @@ protocol Tokenable {
 extension STPToken: Tokenable { }
 
 protocol Clientable {
-    func createToken(withCard card: STPCardParams, completion: ((Tokenable?, Error?) -> Void)?)
+    func kiosk_createToken(withCard card: STPCardParams, completion: ((Tokenable?, Error?) -> Void)?)
 }
 extension STPAPIClient: Clientable {
-    func createToken(withCard card: STPCardParams, completion: ((Tokenable?, Error?) -> Void)?) {
+    func kiosk_createToken(withCard card: STPCardParams, completion: ((Tokenable?, Error?) -> Void)?) {
```
Comment (ashfurrow) [inline Kiosk/Bid Fulfillment/StripeManager.swift]: So naming the protocol method the same as the actual method caused some confusion, and an infinite recursion loop 😒

File: Kiosk/Auction Listings/ListingsViewController.swift [hunk 2]
```diff
@@ -122,6 +122,7 @@ class ListingsViewController: UIViewController {
             .subscribe(onNext: { [weak self] layout in
                 // Need to explicitly call animated: false and reload to avoid animation
                 self?.collectionView.setCollectionViewLayout(layout, animated: false)
+                self?.collectionView.reloadData()
```
Comment (ashfurrow) [inline Kiosk/Auction Listings/ListingsViewController.swift]: Don't know what went wrong here, but calling `reloadData()` fixed it (like most things in UICollectionView).

**Comment by orta** [comment]
assigneeeeessss ash ;)

File: Kiosk/Auction Listings/ListingsViewController.swift (hunk 2 above)
Comment (orta) [inline Kiosk/Auction Listings/ListingsViewController.swift]: yeah, sounds legit, I remember having issues with this building Eigens waterfall layouts

File: Kiosk/Bid Fulfillment/StripeManager.swift (hunk 1 above)
Comment (orta) [inline Kiosk/Bid Fulfillment/StripeManager.swift]: Tricksy

File: Kiosk/Storyboards/Fulfillment.storyboard [hunk 3]
```diff
@@ -414,10 +414,10 @@
                                         </connections>
                                     </button>
                                     <button opaque="NO" contentMode="scaleToFill" fixedFrame="YES" contentHorizontalAlignment="center" contentVerticalAlignment="center" lineBreakMode="middleTruncation" translatesAutoresizingMaskIntoConstraints="NO" id="4eS-5T-1qw" customClass="ActionButton" customModule="Kiosk" customModuleProvider="target">
-                                        <rect key="frame" x="263" y="17" width="81" height="43"/>
+                                        <rect key="frame" x="210" y="17" width="109" height="43"/>
                                         <autoresizingMask key="autoresizingMask" flexibleMaxX="YES" flexibleMaxY="YES"/>
                                         <color key="backgroundColor" red="0.95686274510000002" green="0.95686274510000002" blue="0.95686274510000002" alpha="1" colorSpace="custom" customColorSpace="sRGB"/>
-                                        <state key="normal" title="CC 👍">
+                                        <state key="normal" title="CC Accept">
```
Comment (orta) [inline Kiosk/Storyboards/Fulfillment.storyboard]: hah

**Pull request merged**